### DISCUSSION TITLE
Logs

### DISCUSSION THREAD

**Discussion by k33g** [vmware-labs/wasm-workers-server#252] [Ideas]
Hello, is it possible to deactivate the print of the logs?

**Comment by Angelmmiguel** [comment]
Hey @k33g! 

Thanks for opening the discussion 😄 . Do you mean all of them, server, request or worker logs? Just to understand your use case, why do you need to deactivate them?

Thank you!

**Reply by k33g** [reply]
@Angelmmiguel I think worker logs
![image](https://github.com/vmware-labs/wasm-workers-server/assets/491848/0fa67bdd-4821-4697-8001-93222eb33360)

I would like to deactivate the output, because, it slows the requests

**Reply by Angelmmiguel** [reply]
I see! Good point :). Those are "request logs" and they are generated by the `actix` library. I believe we can provide different options to use different formats like:

```
wws --request-logs [full|minimal|off] --port 3000 .
``` 

I'm gonna create a separate task for it. 

Thanks for the suggestion!

**Reply by Angelmmiguel** [reply]
@assambar created a task to cover this @k33g 😄 https://github.com/vmware-labs/wasm-workers-server/issues/253
